src/main/java/bio/terra/workspace/service/workspace/flight/CreateGoogleContextRBSFlight.java

```diff
@@ -1,24 +1,44 @@
 package bio.terra.workspace.service.workspace.flight;
 
+import bio.terra.cloudres.google.billing.CloudBillingClientCow;
+import bio.terra.cloudres.google.cloudresourcemanager.CloudResourceManagerCow;
 import bio.terra.stairway.Flight;
 import bio.terra.stairway.FlightMap;
 import bio.terra.stairway.RetryRule;
 import bio.terra.stairway.RetryRuleExponentialBackoff;
+import bio.terra.workspace.db.WorkspaceDao;
 import bio.terra.workspace.service.buffer.BufferService;
 import org.springframework.context.ApplicationContext;
+import org.springframework.transaction.support.TransactionTemplate;
 
 // NOTE: DO NOT USE. Currently just a shell class to exercise connection to Buffer Service.
 public class CreateGoogleContextRBSFlight extends Flight {
   public CreateGoogleContextRBSFlight(FlightMap inputParameters, Object applicationContext) {
     super(inputParameters, applicationContext);
     ApplicationContext appContext = (ApplicationContext) applicationContext;
     BufferService bufferService = appContext.getBean(BufferService.class);
-    // TODO(tovanadler): Update the retry rules once we properly design the flight.
-    RetryRule retryRule =
+    CloudResourceManagerCow resourceManager = appContext.getBean(CloudResourceManagerCow.class);
+    CloudBillingClientCow billingClient = appContext.getBean(CloudBillingClientCow.class);
+    WorkspaceDao workspaceDao = appContext.getBean(WorkspaceDao.class);
+    TransactionTemplate transactionTemplate = appContext.getBean(TransactionTemplate.class);
+
+    RetryRule rbsRetryRule =
+        new RetryRuleExponentialBackoff(
+            /* initialIntervalSeconds= */ 1,
+            /* maxIntervalSeconds= */ 5 * 60,
+            /* maxOperationTimeSeconds= */ 20 * 60);
+
+    RetryRule storeRetryRule =
         new RetryRuleExponentialBackoff(
             /* initialIntervalSeconds= */ 1,
-            /* maxIntervalSeconds= */ 5*60,
+            /* maxIntervalSeconds= */ 8,
             /* maxOperationTimeSeconds= */ 16);
-    addStep(new PullProjectFromPoolStep(bufferService), retryRule);
+
+    addStep(new GenerateResourceIdStep());
+    addStep(new PullProjectFromPoolStep(bufferService, resourceManager), rbsRetryRule);
+
+    // TODO(tovanadler): Enable this once we fix configurations.
+    // addStep(new SetProjectBillingStep(billingClient));
+    // addStep(new StoreGoogleContextStep(workspaceDao, transactionTemplate), storeRetryRule);
```

Comment:
It looks like StoreGoogleContextStep needs a workspace ID in the flight input, this will cause your test to fail for now

Reply:
Yup, just commented it out. Meant to before.

Reply:
jk, looks like you beat me to it